app/grandchallenge/profiles/templates/profiles/subscription_form.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+{% load static %}
+{% load crispy_forms_tags %}
+
+
+{% block content %}
+    <h2>Subscriptions</h2>
+    <p>Your current subscription preferences are as follows:</p>
+    {% csrf_token %}

```

Comment:
Just for my understanding - I thought that crispy included the csrf token?

Reply:
Also correct. In a first implementation I didn't use `{% crispy form %}` and when changing it I forgot to remove the csrf template tag.